src/formatDate.js

```diff
@@ -8,7 +8,38 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
```

Comment:
The use of Array.prototype.pop() here is mutating the original `fromFormat` array, which could lead to unexpected behavior if the array is used elsewhere after calling this function. Consider using a different approach to access the last element without mutating the array.

Reply:
Avoid arguments mutation